### PR TITLE
format form for firefox and reconnect to game

### PR DESCRIPTION
**Pull request by jessebmiller**
Updates format for firefox

image

### CODE REVIEW

**Comment by jessebmiller** [comment]
I think I mitigated the dropping the game issues well enough for people to play consistently

File: packages/app-spec-squad-chess/src/Matchmaker.js [hunk 1]
```diff
@@ -265,9 +262,33 @@ const handleSendAnswer = (event) => {
   m.redraw()
 }
 
+export const sendMessage = (message) => {
```
Comment (EzraWeller) [inline packages/app-spec-squad-chess/src/Matchmaker.js:265]: Is there a reason this is in the "handlers" section? Can we move it into the main functions section?

**Review by EzraWeller — CHANGES_REQUESTED** [review]
Looks good to me.

File: packages/app-spec-squad-chess/src/Matchmaker.js (hunk 1 above)
Comment (jessebmiller) [reply packages/app-spec-squad-chess/src/Matchmaker.js:265]: Ah sure that would be fine. I moved it here so I could see it next to it's handler as I worked on them both so I didn't have to jump back and forth keeping them in sync with each other.

Comment (EzraWeller) [reply packages/app-spec-squad-chess/src/Matchmaker.js:265]: Oh, woops, this is the mithril app -- I thought this was the send message in the matchmaking client when I wrote this. This is actually the only non-component non-handler function in this file, I think, so it's fine. Let's merge!

**Review by EzraWeller — APPROVED** [review]

**Pull request merged**